### DISCUSSION TITLE
WPF - Could not find assembly CefSharp during compile

### DISCUSSION THREAD

**Discussion by jeanmichelmorin** [cefsharp/CefSharp#4313] [Q&A]
When I try to compile it, this is what I get:

Error MC1000	Unknown Build Error, 'Could not find assembly 'CefSharp, Version=107.1.90.0, Culture=neutral, PublicKeyToken=40c4b6fc221f4138'. Either explicitly load this assembly using a method such as LoadFromAssemblyPath() or use a MetadataAssemblyResolver that returns a valid assembly. Line 40 Position 38.' ../page1.xaml

![image](https://user-images.githubusercontent.com/44597109/203134422-0cd632ce-52fa-4deb-89e7-61629238b54f.png)

**Comment by amaitland** [comment]
Best guess on the very limited detail provided is that you have installed the incorrect Nuget package for the .Net version you are using. 

The [Quick Start](https://github.com/cefsharp/CefSharp/wiki/Quick-Start) has different instructions based on the .Net version. Make sure you install the correct package.
